mlfinlab/tests/test_backtest_statistics.py

```diff
@@ -0,0 +1,202 @@
+"""
```

Comment:
Are we also testing for edge cases?

Reply:
I can add more examples. By edge cases you mean something like nan values or just higher range of test values?

Reply:
Yea testing for things like nans or any other strange values that could be passed to the data. 

Does the function opperate as expected under edge case examples.

Reply:
I haven't seen such explicit tests for function in other files. I would like to know how deep the testing should go?

Reply:
The tests @Aditya wrote for the portfolio optimisation module sets the best example. 

For several of our modules we wrote tests that arent complete and I want to make sure we correct that mistake going forward.

Reply:
@Jackal08 Wrong Aditya mentioned. Its @aditya1702 😅

Reply:
@PanPip You can find the tests in the file test_portfolio_optimisation.py

Reply:
@aditya1702 I can't see explicit tests for NaNs in that file

Reply:
@PanPip Ah, yes I have not written tests for NaNs. I meant being more stringent in coverage of your code through the tests. For example if there are parts of code which require an error to be thrown then you need to write tests to check if an error is thrown properly or not.

However,  gave your tests a look and they do seem good to me 👍

Reply:
@aditya1702  Thank you :)